examples/ui-demo/src/components/content/ContentNav.tsx

```diff
@@ -0,0 +1,57 @@
+import { cn } from "@/lib/utils";
+import { RenderUserConnectionAvatar } from "../user-connection-avatar/RenderUserConnectionAvatar";
+import { useUser } from "@account-kit/react";
+import ExternalLink from "@/components/shared/ExternalLink";
+import { Metrics } from "@/metrics";
+import { CodePreviewSwitch } from "@/components/shared/CodePreviewSwitch";
+export function ContentNav({
```

Comment:
if `ContentWrapper` is changed to `PreviewWrapper`, this should also be `PreviewNav`!